packages/react-core/src/components/TreeView/examples/TreeView.md

```diff
@@ -14,23 +14,30 @@ import { FolderIcon, FolderOpenIcon, EllipsisVIcon, ClipboardIcon, HamburgerIcon
 
 ```js
 import React from 'react';
-import { TreeView, TreeViewDataItem } from '@patternfly/react-core';
+import { TreeView, TreeViewDataItem, Button } from '@patternfly/react-core';
 
 class DefaultTreeView extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = { activeItems: {} };
+    this.state = { activeItems: {}, allExpanded: null };
```

Comment:
Why `null`?

Reply:
`null` allows the `defaultExpanded` prop to function properly (if you don't want to force the state for everything before you start filtering/etc). It can be set to `true` or `false` if `defaultExpanded` is not used or it is fine to override it.